readthedocs/builds/tasks.py

```diff
@@ -178,67 +178,61 @@ def _get_version(self, task, args, kwargs):
         return version
 
 
-class ArchiveBuilds(Task):
-
-    """Task to archive old builds to cold storage."""
-
-    name = __name__ + '.archive_builds'
-
-    def run(self, *args, **kwargs):
-        if not settings.RTD_SAVE_BUILD_COMMANDS_TO_STORAGE:
-            return
-
-        lock_id = '{0}-lock'.format(self.name)
-        days = kwargs.get('days', 14)
-        limit = kwargs.get('limit', 2000)
-        delete = kwargs.get('delete', True)
-
-        with memcache_lock(lock_id, self.app.oid) as acquired:
-            if acquired:
-                archive_builds_task(days=days, limit=limit, delete=delete)
-            else:
-                log.warning('Archive Builds Task still locked')
+# NOTE: re-define this task to keep the old name. Delete it after deploy.
+@app.task(queue='web')
+def archive_builds(days=14, limit=200, include_cold=False, delete=False):
+    archive_builds_task.delay(days=days, limit=limit, include_cold=include_cold, delete=delete)
 
 
-def archive_builds_task(days=14, limit=200, include_cold=False, delete=False):
+@app.task(queue='web', bind=True)
+def archive_builds_task(self, days=14, limit=200, include_cold=False, delete=False):
```

Comment:
I think the old name is fine, we're also losing the previous defaults here, we should change them here or change them on the celery beat settings.

Reply:
I just followed the convention that we have for other tasks that are function, where we postfix them with `_task`.

Regarding the defaults, I've updated them in the celery beat setting 👍🏼

Reply:
> I just followed the convention that we have for other tasks that are function

We have a mix of both, so not really sure if we're following a convention (but personally adding _task is redundant for me).

And if you rename this task, it should be fine to not keep the old name around, since this isn't a task that is triggered by a user action.